bluemira/equilibria/find.py

```diff
@@ -725,6 +728,90 @@ def get_flux_loop(psi_norm):
     return lcfs, separatrix
 
 
+def get_legs(equilibrium):
+    """
+    Get the legs of a separatrix.
+
+    Parameters
+    ----------
+    equilibrium: Equilibrium
+        Equilibrium for which to find the separatrix legs
+
+    Returns
+    -------
+    legs: List[Loop]
+        Geometries of the legs, sorted as:
+            [lower inner,  # if present
+             lower outer,  # if present
+             upper inner,  # if present
+             upper outer,  # if present
+            ]
+
+    Raises
+    ------
+    EquilibriaError: if a strange number of legs would be found for an X-point
+
+    Notes
+    -----
+    Will return two legs in the case of a single null
+    Will return four legs in the case of a double null
+    """
+
+    def extract_leg(flux_line, x_point):
+        radial_line = Loop(
+            x=[x_point.x - delta, x_point.x + delta], z=[x_point.z, x_point.z]
+        )
+        arg_inters = join_intersect(flux_line, radial_line, get_arg=True)
+        arg_inters.sort()
+        if x_point.z < o_p.z:
+            # Lower null
+            func = operator.lt
+        else:
+            # Upper null
+            func = operator.gt
```

Comment:
```suggestion
        # Lower null == Lessthan,  Upper null == Morethan
        func = operator.lt if x_point.z < o_p.z else operator.gt
```
as the if else here is just defining a single short variable